lib/protocol/apdu.js

```diff
@@ -205,14 +226,20 @@ class APDUCommand {
 
   /**
    * Get wallet public key
-   * @param {Number[]}
+   * @param {Number[]} path
+   * @param {apdu.addressFlags} [addressFlags=0]
    * @returns {APDUCommand}
    */
 
-  static getPublicKey(path) {
+  static getPublicKey(path, addressFlags) {
     const data = encodePath(path);
 
     return new APDUCommand({
+      // verify ?
```

Comment:
why the question mark?

Reply:
Whether we should verify or not is decided by that flag, should I remove it ? :)